indico/web/client/styles/design_system/_index.scss

```diff
@@ -0,0 +1,15 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2023 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+@use 'variables';
+@use 'button';
+@use 'button_group';
+@use 'form_controls';
+@use 'layout';
+@use 'popup';
+@use 'visibility';
+@use 'utils';
```

Comment:
I'd be clear using `./whatever` that those are relative imports and not some top-level thing (like `'base'` would be for example)